.github/workflows/version.yml

```diff
@@ -27,13 +33,17 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ env.BASE }}
+          token: ${{ steps.github_app_token.outputs.token }}
```

Comment:
Curious, if we don't give the app token here, what things it could break, will it cause the new PR not triggering CI?

Reply:
Do you mean for this specific step or overall for using the token?

Reply:
Not overall, just for this checkout action. If this checkout still use the default token, we only pass this app token to the next create PR action, will it cause the new PR not triggering CI?

Reply:
I think I have seen multiple recommendations from this GHA to use the same token in actions/checkout as well. One of them is https://github.com/peter-evans/create-pull-request/issues/990